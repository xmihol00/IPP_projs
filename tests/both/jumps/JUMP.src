.IPPcode21

WRITE string@jumping\032at\032label1\010
JUMP label1

WRITE string@jump\0321\032failed\010
LABEL label1
WRITE string@label1\032here\010

JUMP skip

LABEL label2

WRITE string@jumping\032back\032down\010
JUMP label3

LABEL skip

WRITE string@jumping\032above\010
JUMP label2

LABEL label3
WRITE string@down\032is\032here\010
