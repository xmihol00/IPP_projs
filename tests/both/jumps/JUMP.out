jumping at label1
label1 here
jumping above
jumping back down
down is here
